webapp/components/Map.tsx

```
@@ -1,27 +1,235 @@
   }, []);
 
-  return <div ref={mapContainer} className="w-full h-[calc(100vh-8rem)]" />;
+  const deckLayers = useMemo(() => {
+    const updateTriggers = {
+      getFillColor: [year, hoveredElement],
+    };
+
+    return [
+      new TileLayer({
+        id: OVERLAY_LAYER,
+        getTileData: tileSource && tileSource.getTileData,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)